twig_server/database/graph/Project.py

```diff
@@ -0,0 +1,4 @@
+from graphobjects import AbstractNode
+
+class Project(AbstractNode):
+    __label_name = "Project"
```

Comment:
newline EOL pls

also btw Project should not be inheriting from AbstractNode it should be inheriting from Node